modules/light-clients/08-wasm/keeper/genesis_test.go

```diff
@@ -57,7 +57,7 @@ func (suite *KeeperTestSuite) TestInitGenesis() {
 			suite.Require().NoError(err)
 
 			var storedHashes []string
-			res := types.GetCodeHashes(suite.chainA.GetContext(), GetSimApp(suite.chainA).AppCodec())
+			res, err := types.GetCodeHashes(suite.chainA.GetContext(), GetSimApp(suite.chainA).AppCodec())
```

Comment:
should we move `suite.Require().NoError(err)` from L65 up to check the error here? 

guess it doesn't make difference actually!